### ISSUE TITLE
Boolean attributes in head config are rendered incorrectly

### ISSUE THREAD

**Issue by trygveaa** [nuxt/nuxt.js#6045]
### Version

[v2.8.1](https://github.com/nuxt.js/releases/tag/v2.8.1)

### Reproduction link

[https://codesandbox.io/s/codesandbox-nuxt-z7sg0](https://codesandbox.io/s/codesandbox-nuxt-z7sg0)

### Steps to reproduce

In the provided repro, look at the rendered `link` and `script` and tags in head. The relevant ones are the ones for `test.css` and `test.js`.

To produce the repro, this was added in nuxt.config.js:
```
    link: [{ rel: "stylesheet", href: "/test.css", disabled: true }],
    script: [{ src: "/test.js", async: false, defer: true }]
```

### What is expected ?

Boolean attributes in HTML5 should be rendered as `name`, `name=""` or `name="name` if true, and not rendered if false.

### What is actually happening?

The boolean attributes `disabled` for link and `async` and `defer` for script are rendered as `disabled="true"`, `async="false"` and `defer="true"`

This is not allowed in HTML5. See https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#boolean-attributes
> The values "true" and "false" are not allowed on boolean attributes. To represent a false value, the attribute has to be omitted altogether.

<div align="right"><sub><em>This bug report is available on <a href="https://cmty.app/nuxt">Nuxt</a> community (<a href="https://cmty.app/nuxt/nuxt.js/issues/c9465">#c9465</a>)</em></sub></div>

**Comment by pimlie** [comment]
This should already be solved in vue-meta v2. You can manually upgrade yourself to v2, Nuxt will print a warning about a not recommended version but you can ignore that.

**Comment by trygveaa** [comment]
Ah, with that version attributes set to true are rendered correctly. Attributes set to false are rendered as enabled though. I don't really have a use case for false values, but it seems a bit strange.

**Comment by pimlie** [comment]
Yeah, thats not supposed to happen ofc. Moving this issue to vue-meta repo :)
